TreeDB/internal/valuelog/reader_mmap.go

```
@@ -1286,7 +1286,7 @@ func (f *File) readViaMmapAppend(ptr page.ValuePtr, verifyCRC bool, dst []byte)
 		return nil, ErrCorrupt, true
 	}
 
-	if cachedVal, usedDst, err, hit := f.groupedFrameCacheReadTo(start, verifyCRC, k, offsets, rawLen, subIndex, dst[len(dst):len(dst)]); hit {
+	if cachedVal, usedDst, err, hit := f.groupedFrameCacheReadTo(start, verifyCRC, k, &offsets, rawLen, subIndex, dst[len(dst):len(dst)]); hit {
```

golangci-lint (2.13.2)

[error] S1010: should omit second index in slice, s[a:len(s)] is identical to s[a:]

(staticcheck)